benchmarks/tasks.py

```diff
@@ -39,32 +78,33 @@ def set_testjob_results(self, testjob):
     if not testjob.initialized:
         testjob.testrunnerclass = tester.get_result_class_name(testjob.id)
         testjob.initialized = True
-        testjob.save()
```

Comment:
I don't understand this bit. If the testjob.initialized isn't saved to db it's lost and the condition 'if not testjob.initialized' will be met as the default is False. Am I missing something here?